grades/models.py

```diff
@@ -141,6 +142,28 @@ def __str__(self):
         )
 
 
+class MicromastersCertificate(TimestampedModel):
+    """
+    Model for storing MicroMasters course certificates
```

Comment:
Since this is only about a grade for a course run the model should probably be `MicromastersCourseCertificate`. Is this certificate independent of exam grades, and overall passing of the course?